apps/tx-builder/src/components/forms/fields/SelectContractField.tsx

```diff
@@ -11,7 +11,10 @@ type SelectContractFieldTypes = {
 const SelectContractField = ({ value, onChange, options }: SelectContractFieldTypes) => {
   return (
     <StyledSelect
+      name="select-contract"
+      label="Select Contract"
```

Comment:
I'm trying to figure out where this component would be used. Just by code review I can't imagine where this label could fit, because we are not selecting a contract anywhere

Reply:
done!

It seems that there is a bug in the components library, @JagoFigueroa created [a ticket to address it](https://github.com/gnosis/safe-react-apps/issues/360)

![Captura de pantalla 2022-03-07 a las 14 22 18](https://user-images.githubusercontent.com/26763673/157056778-2176aa61-7bf5-493a-bd26-b38d380b47d7.png)